frontend/assets/css/style.css

```diff
@@ -0,0 +1,278 @@
+}
+
+
+/* .nav-icon {
+    width: 64px;
+    height: 64px;
+} */
+
+#inner,
+#inner-blue {
+    display: table;
+    width: 5%;
+    margin: 0 auto;
+    /*
+    background-color: #4CAF50;
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 4px 2px;
+    cursor: pointer;
+    */
+}
+
+div .background {
+    -webkit-box-shadow: 3px 4px 5px -2px rgba(0, 0, 0, 0.4);
+    -moz-box-shadow: 3px 4px 5px -2px rgba(0, 0, 0, 0.4);
+    box-shadow: 3px 4px 5px -2px rgba(0, 0, 0, 0.4);
+}
+
+.background {
+    background: url(../img/student.jpg);
+    background-size: cover;
+    height: 200px;
+    width: 200px;
+    border-radius: 10px;
+}
+
+.background-res {
+    background-size: cover;
+    height: 100px;
+    width: 200px;
+    border-radius: 10px;
+}
+
+.transbox {
+    height: 100%;
+    width: 100%;
+    opacity: 0.8;
+    filter: alpha(opacity=60);
+    text-align: center;
+    padding: 15px;
+    padding-top: 25px;
+    border-radius: 10px;
+}
+
+.transbox p {
+    margin: 5px;
+    font-size: 15px;
+    letter-spacing: 1px;
+    color: #ffffff;
+}
+
+.blue {
+    background-color: #08b2ea;
+}
+
+.green {
+    background-color: rgba(80, 210, 104, 0.98);
+}
+
+.dark {
+    background-color: #34444A;
+}
+
+.dark-res {
+    background-color: #082029;
+}
+
+#sec-cat,
+#grad-awt-cat,
+#empl-cat {
+    margin: 10px;
+}
+
+#inner a:link,
+#inner a:visited {
+    /* background-color: rgba(80, 210, 104, 0.98);
+    padding: 14px 25px; */
+    color: white;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    background-color: #31AD74;
+    padding: 10px 30px;
+    border-radius: 2px;
+    letter-spacing: 1.2px;
+}
+
+#inner a:hover,
+#inner a:active {
+    background-color: #44B47E;
+}
+
+#inner-blue a:link,
+#inner-blue a:visited {
+    /* background-color: rgba(80, 210, 104, 0.98);
+    padding: 14px 25px; */
+    color: white;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    background-color: #66A1E8;
+    padding: 10px 30px;
+    border-radius: 2px;
+    letter-spacing: 1.2px;
+}
+
+#inner-blue a:hover,
+#inner-blue a:active {
+    background-color: #9188F0;
+}
+
+.sec {
+    font-weight: 500;
+    color: #777;
+    font-family: "apple-system", "BlinkMacSystemFont", "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", "sans-serif";
+}
+
+h2,
+h3,
+label {
+    font-weight: 500;
+    /* color: #777; */
+    font-family: "apple-system", "BlinkMacSystemFont", "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", "sans-serif";
+}
+
+.center-div {
+    position: absolute;
+    margin: auto;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: #ccc;
+    border-radius: 3px;
+}
+
+.questions-bx,
```

Comment:
make the box look like a card, add box shadow to it, and padding (the content are too close to the borders)